doc/Requirements Specification/Parts/Purpose.lyx

```diff
@@ -398,7 +398,7 @@ gls{SoMoX}
 \end_inset
 
 .
- It aims to conduct performance measurements on source code of a software,
+ It aims to conduct performance measurements on source code of a software
```

Comment:
“a software”! Maybe use “It aims to conduct performance measurements on software’s source code”?